website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Materialize queries with exports"
+description: "Use exports to materialize tables to the data platform on a schedule."
+sidebar_label: "Materialize with exports"
+---
+
+Exports with the dbt Semantic Layer enhance the [saved queries](/docs/build/saved-queries) feature, allowing you to materialize commonly used queries directly within your data platform.
+
+While saved queries are a way to save and reuse commonly used queries in MetricFlow, exports take this functionality a step further by:
+
+- Enabling you to materialize these queries within your data platform using dbt Cloud.
+- Proving an integration path for tools that don't natively support the dbt Semantic Layer by exposing tables of metrics and dimensions.
+
+Essentially, exports are like any other table in your data platform. They allow you to query metric definition through any SQL interface or connect to downstream tools without needing a first class Semantic Layer integration. Refer to [Available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) for more information.
+
+## Prerequisites
+
+- You have a dbt Cloud account on a [Team or Enterprise](https://www.getdbt.com/pricing/) plan.
+- You are on [dbt version](/docs/dbt-versions/upgrade-core-in-cloud) 1.7 or higher.
```

Comment:
sugg: i was taught that newer is preferred to higher bc it's more translation friendly (ie "higher" can have diff meanings depending on context)  

```suggestion
- You are on [dbt version](/docs/dbt-versions/upgrade-core-in-cloud) 1.7 or newer.
```

Reply:
i think we've been using 'higher' more but i think we should chat about our stance on this again. will put it in the agenda! 

i prefer 'higher' bc it indicates that the version number must be greater than the specified version and bc newer versions are generally numerically higher, it's understood that higher numbers mean more recent releases. but there's more to this now i think - so worth a quick chat for sure!